### DISCUSSION TITLE
Error when linking to Resideo Server

### DISCUSSION THREAD

**Discussion by wredd09** [d0n4v4nb3ck3r/homebridge-honeywell-home#775] [General]
Has anyone had issues linking an account to the Honeywell servers? I get through configuration and the sign in with them, but after selecting my thermostats to add and hitting connect, I receive an error 35, revocation function unable to check. Error below with some details changed to XXXXX. I've tried older versions, and the beta and receive the same error. 

Thanks in advance! 

An error occurred:
{"code":35,"killed":false,"signal":null,"cmd":"curl -X POST --header \"Authorization: Basic XXXXXXXXXXXXXXXXXXXX==\" --header \"Accept: application/json\" --header \"Content-Type: application/x-www-form-urlencoded\" -d \"grant_type=authorization_code&code=XXXXXX&redirect_uri=http://192.168.1.XXXXX/auth\" \"https://api.honeywell.com/oauth2/token\"","stdout":"","stderr":" % Total % Received % Xferd Average Speed Time Time Time Current\r\n Dload Upload Total Spent Left Speed\r\n\r 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0\r 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0\r 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0\r\ncurl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.\r\n"}

Close this window and start again

**Comment by mnich0ls** [comment]
Are you using the callback URL from the first step? http://homebridge.local:8585/auth

I see an IP address in the redirect URL, but I'm not sure if that's actually the issue. Just a guess.

**Reply by mnich0ls** [reply]
Your port may be different than mine. I'm not sure how the port is determined e.g. 8585

**Reply by donavanbecker** [reply]
Yep 8585 is the default port that I use.

**Reply by wredd09** [reply]
The URL that the setup provides isn't homebridge.local, but rather the specific IP of the computer running homebridge. Can't imagine that makes a difference since it should point to the same place, but I'll try that too. What's interesting is that it is not my homebridge port; however I could not find any more info as to whether that should be the same or not.

**Reply by wredd09** [reply]
Update, changing it to .local resulted in error - {"status":400,"message":"Error generating auth code: undefined","type":"internal"}

**Comment by wredd09** [comment]
Still going on. Any other thoughts on how to fix or bypass the process are welcome.
